### PR TITLE
bitnami/dataplatform-bp1: added annotation based discovery feature in wavefront to the chart

### PR DESCRIPTION
**Pull request by sarvani95**
**Description of the change**
Updated  wavefront to use auto discovery feature to the Dataplatform chart

**Benefits**
Wavefront will be able to fetch metrics from dataplatform applications/pod automatically using annotation based discovery.

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

File: bitnami/dataplatform-bp1/Chart.yaml [hunk 1]
```diff
@@ -1,62 +1,61 @@
 annotations:
   category: Infrastructure
 apiVersion: v2
-appVersion: 0.0.11
+appVersion: 1.0.0
```
Comment (gkulkarni32) [inline bitnami/dataplatform-bp1/Chart.yaml:4]: update this to the dataplatform-exporter version

**Review by gkulkarni32 — CHANGES_REQUESTED** [review]
All the comments for bp2 PR are applicable here too (except the app version which is the exporter app version in case of bp1).
https://github.com/bitnami/charts/pull/8064#pullrequestreview-800892104

**Comment by rafariossaa** [comment]
Hi,
  Thanks for the PR.
  Could you take a look to the linter errors ?

**Comment by sarvani95** [comment]
Hi @rafariossaa,

Could you please tell me the linter errors you found, I am not able to run the workflow for Lint and test charts, it says First-time contributors need a maintainer to approve running workflows.

File: bitnami/dataplatform-bp1/values.yaml [hunk 2]
```diff
@@ -1002,61 +1106,60 @@ wavefront:
       enableRuntimeConfigs: true
       ## @param wavefront.collector.discovery.config [array] Configuration for rules based auto-discovery
       ##
-      config:
+      #config:
```
Comment (rafariossaa) [inline bitnami/dataplatform-bp1/values.yaml:1109]: The linter complains about a missing space here:
```suggestion
      # config:
```

**Review by rafariossaa — CHANGES_REQUESTED** [review]

**Comment by rafariossaa** [comment]
Hi,
  This is the log from the linter:
```Linting chart 'dataplatform-bp1 => (version: "9.0.0", path: "bitnami/dataplatform-bp1")'
Checking chart 'dataplatform-bp1 => (version: "9.0.0", path: "bitnami/dataplatform-bp1")' for a version bump...
>>> git cat-file -e origin/master:bitnami/dataplatform-bp1/Chart.yaml
>>> git show origin/master:bitnami/dataplatform-bp1/Chart.yaml
Old chart version: 8.0.2
New chart version: 9.0.0
Chart version ok.
>>> yamale --schema /opt/hostedtoolcache/ct/v3.4.0/x86_64/etc/chart_schema.yaml bitnami/dataplatform-bp1/Chart.yaml
Validating /home/runner/work/charts/charts/bitnami/dataplatform-bp1/Chart.yaml...
Validation success! 👍
>>> yamllint --config-file /opt/hostedtoolcache/ct/v3.4.0/x86_64/etc/lintconf.yaml bitnami/dataplatform-bp1/Chart.yaml
>>> yamllint --config-file /opt/hostedtoolcache/ct/v3.4.0/x86_64/etc/lintconf.yaml bitnami/dataplatform-bp1/values.yaml
Error: [comments] missing starting space in comment
Error: Error linting charts: Error processing charts
```
It is just a missing space in `values.yaml`

**Comment by rafariossaa** [comment]
Thanks for fixing it.
I am merging the changes.

**Pull request merged**